### PR TITLE
Edit fitle

### PR DESCRIPTION
**Pull request by Rana29hesham**

### CODE REVIEW

File: newFile.txt [hunk 1]
```diff
@@ -1 +1,4 @@
 New file on Feature1 branch
+
+
```
Comment (KareemZafan) [inline newFile.txt]: No need for white spaces

**Review by KareemZafan — CHANGES_REQUESTED** [review]

**Review by Abdelrahmanbeshiir — CHANGES_REQUESTED** [review]
need improvement

File: newFile.txt [hunk 2]
```diff
@@ -1 +1,4 @@
 New file on Feature1 branch
+
```
Comment (ghost) [inline newFile.txt]: Remove any spaces

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by Abdelrahmanbeshiir — APPROVED** [review]
well done

**Review by ghost — APPROVED** [review]
Well done!

**Review by KareemZafan — APPROVED** [review]

**Pull request merged**